py/private/py_binary.bzl

```diff
@@ -139,6 +139,41 @@ _attrs = dict({
         allow_single_file = True,
         mandatory = True,
     ),
+    "python_version": attr.string(
+        doc = """Whether to build this target and its transitive deps for a specific python version.
+
+Note that setting this attribute alone will not be enough as the python toolchain for the desired version
+also needs to be registered in the WORKSPACE or MODULE.bazel file.
+
+When using WORKSPACE, this may look like this,
+
+```
+load("@rules_python//python:repositories.bzl", "py_repositories", "python_register_toolchains")
+
+python_register_toolchains(
+    name = "python_toolchain_3_8",
+    python_version = "3.8.12",
+    # setting set_python_version_constraint makes it so that only matches py_* rule  
```

Comment:
Can this be updated to?

Reply:
i don't follow.

Reply:
do you mean, can it be updated to some other version?

Reply:
It's the similar comment that I didn't understand from above, so if that one is reworded, can this one be updated too?

Reply:
done